bindata/oauth-apiserver/authenticator-clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:openshift:tokenvalidator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:openshift:tokenvalidator
+subjects:
+- kind: ServiceAccount
+  namespace: openshift-oauth-apiserver
+  name: openshift-authenticator
```

Comment:
this surprises me a little bit. why do we need to have permissions to hit this endpoint?

Reply:
It's an endpoint as any other, this is kube's `system:auth-delegator` w/o the SARs.